administrator/language/de-DE/plg_system_webauthn.ini

```
@@ -19,17 +19,21 @@ PLG_SYSTEM_WEBAUTHN_ERR_CREDENTIAL_ID_ALREADY_IN_USE="Die Anmeldedaten können n
 PLG_SYSTEM_WEBAUTHN_ERR_EMPTY_USERNAME="Bitte nur den Benutzernamen (aber NICHT das Passwort) eingeben, bevor die Anmeldeschaltfläche für die Web-Authentifizierung ausgewählt wird."
 PLG_SYSTEM_WEBAUTHN_ERR_INVALID_USERNAME="Der angegebene Benutzername entspricht nicht einem Benutzerkonto, das eine passwortlose Anmeldung auf dieser Website ermöglicht hat."
 PLG_SYSTEM_WEBAUTHN_ERR_LABEL_NOT_SAVED="Der neue Name konnte nicht gespeichert werden"
+PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Der Authentifizierer konnte nicht entfernt werden"
 PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für passwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle, welche der Browser vertraut, signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
 PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Die gespeicherten Anmeldedaten für diesen Login-Authentifizirer können nicht gefunden werden."
-PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Der Authentifizierer konnte nicht entfernt werden"
 PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="Der Benutzer für diesen Authentifizierer scheint auf dieser Website nicht mehr zu existieren."
+PLG_SYSTEM_WEBAUTHN_ERR_XHR_INITCREATE="Die Registrierungsinformationen des Authentifizierers können nicht von der Website abgerufen werden."
+PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_DESC="Nur Authentifizierer mit überprüfbaren kryptografischen Signaturen für WebAuthn-Anmeldungen zulassen. Für Hochsicherheitsumgebungen dringend empfohlen. Erfordert, dass die Website direkt auf <code>https://mds.fidoalliance.org/</code> zugreifen kann, ein beschreibbares Cache-Verzeichnis, das temporäre Systemverzeichnis, welches von PHP beschreibbar ist, und die OpenSSL-Erweiterung. Kann einige billigere, nicht zertifizierte Authentifizierer daran hindern, überhaupt zu funktionieren. Das Deaktivieren hindert Joomla auch daran, Marke und Modell des verwendeten Authentifizierers zu identifizieren (neben dem Authentifikatornamen wird kein Symbol angezeigt).<br/><strong>Profi-Tipp:</strong> Wenn man sich hinter einer Firewall befindet, so können die von <a href='https://mds.fidoalliance.org/' target='_blank'>der FIDO-Allianz</a> heruntergeladenen Daten in die Datei <code>administrator/cache/fido.jwt</code> speichern, damit diese Funktion richtig funktioniert."
+PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_LABEL="Attestation Support"
```

```suggestion
PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_LABEL="Hilfe beim Attestieren"
```
oder auch "Hilfe beim Beglaubigen"